apps/commandLineApplication.h

```diff
@@ -0,0 +1,235 @@
+              "    fivox[compartments]://BlueConfig?report=string&target=string\n"
+              "- Soma reports:\n"
+              "    fivoxsomas://BlueConfig?report=string&target=string\n"
+              "- Spike reports:\n"
+              "    fivoxspikes://BlueConfig?duration=float&spikes=path&target=string\n"
+              "- Synapse densities:\n"
+              "    fivoxsynapses://BlueConfig?target=string\n"
+              "- Voltage-sensitive dye reports:\n"
+              "    fivoxvsd://BlueConfig?dyecurve=string&target=string\n"
+              "\n"
+              "Parameters for all types :\n"
+              "- BlueConfig: BlueConfig file path\n"
+              "              (default: BBPTestData)\n"
+              "- target: name of the BlueConfig target (default: CircuitTarget)\n"
+              "- inputMin/inputMax: minimum and maximum input values to be considered for rescaling\n"
+              "                     (defaults: [0.0, 2.0] for Spikes and Synapses\n"
+              "                                [-190.0, 0.0] for Compartments with TestData, [-80.0, 0.0] otherwise\n"
+              "                                [-15.0, 0.0] for Somas with TestData, [-80.0, 0.0] otherwise\n"
+              "                                [-0.0000147, 0.00225] for LFP with TestData, [-10.0, 10.0] otherwise\n"
+              "                                [-100000.0, 300.0] for VSD)\n"
+              "- functor: type of functor to sample the data into the voxels\n"
+              "             (defaults: \"density\" for Synapses,\n"
+              "                        \"frequency\" for Spikes,\n"
+              "                        \"field\" for Compartments, Somas and VSD)\n"
+              "- resolution: number of voxels per micrometer (default: 10.0)\n"
+              "- maxBlockSize: maximum memory usage allowed for one block in bytes\n"
+              "                (default: 64MB)\n"
+              "- cutoff: the cutoff distance in micrometers (default: 100).\n"
+              "- extend: the additional distance, in micrometers, by which the\n"
+              "          original data volume will be extended in every dimension\n"
+              "          (default: 0, the volume extent matches the bounding box\n"
+              "          of the data events). Changing this parameter will result\n"
+              "          in more volumetric data, and therefore more computation\n"
+              "          time\n"
+              "- showProgress: display progress bar for current voxelization step\n"
+              "                (default: 0/off)\n"
+              "\n"
+              "Parameters for Compartments:\n"
+              "- report: name of the compartment report\n"
+              "          (default: 'voltage'; 'allvoltage' if BlueConfig is BBPTestData)\n"
+              "- dt: timestep between requested frames in milliseconds\n"
+              "      (default: report dt)\n"
```

Comment:
You haven't mentioned anywhere in the spec that this parameter doesn't have any effect in the new tool.